invokeai/frontend/web/src/features/gallery/components/ImageGrid/GalleryImageGrid.tsx

```diff
@@ -126,7 +126,7 @@ const GalleryImageGrid = () => {
           loadingText={t('gallery.loading')}
           flexShrink={0}
         >
-          {`Load More (${currentData.ids.length} of ${currentViewTotal})`}
+          {`${t('accessibility.loadMore')} (${currentData.ids.length} of ${currentViewTotal})`}
```

Comment:
This isn't quite right - the word "of" will still be used. Maybe the easiest fix is to use a "/" instead?

Otherwise, we'd need to update this translation to use interpolation. But I'm not sure that the "quantity OF total" structure even exists in all languages. So I think just using a "/" is best.

Reply:
done